proxy/src/main/java/com/wavefront/agent/listeners/WavefrontPortUnificationHandler.java

```diff
@@ -116,7 +117,8 @@ public WavefrontPortUnificationHandler(
       @Nullable final SharedGraphiteHostAnnotator annotator,
       @Nullable final Supplier<ReportableEntityPreprocessor> preprocessor,
       final Supplier<Boolean> histogramDisabled, final Supplier<Boolean> traceDisabled,
-      final Supplier<Boolean> spanLogsDisabled, final SpanSampler sampler) {
+      final Supplier<Boolean> spanLogsDisabled, final SpanSampler sampler,
+      final boolean sourceTagsDisabled) {
```

Comment:
as discussed, let's use in-memory queue (capped) instead of passing the sourceTagsDisabled flag